docs/Learning-Environment-Examples.md

```diff
@@ -280,7 +280,7 @@ you would like to contribute environments, please see our
 - Goal:
   - Get the ball into the opponent's goal while preventing the ball from
     entering own goal.
-- Agents: The environment contains four agents, with the same Behavior
+- Agents: The environment contains two different Multi Agent Groups with two agents in each.
```

Comment:
I'm not sure we have to call out explicitly the implementation here. We can just say "The environment contains two opposing groups of agents, each with two agents."